From 51a351080ea6497a3cc7fe08175fe689a23f6ab4 Mon Sep 17 00:00:00 2001
From: Liu Ying <b17645@freescale.com>
Date: Tue, 17 Mar 2009 14:21:27 +0800
Subject: [PATCH 081/449] ENGR00109658 V4l2 capture: Modify rotation IOCTL ID
 for preview task

Rotation IOCTL ID for preview task need to be modify according to the
separation of preview and capture rotation IOCTL ID in kernel.

Signed-off-by: Liu Ying <b17645@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_overlay.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_overlay.c b/test/mxc_v4l2_test/mxc_v4l2_overlay.c
index 433ff3b..b657ba6 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_overlay.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_overlay.c
@@ -181,7 +181,7 @@ mxc_v4l_overlay_setup(struct v4l2_format *fmt)
                 return TFAIL;
         }
 
-        ctl.id = V4L2_CID_PRIVATE_BASE;
+	ctl.id = V4L2_CID_PRIVATE_BASE + 2;
 		ctl.value = g_rotate;
         if (ioctl(fd_v4l, VIDIOC_S_CTRL, &ctl) < 0)
         {
-- 
1.8.0

